src/lib/deploy.js

```diff
@@ -85,7 +85,11 @@ async function deploy({ alias, yes }) {
   }
 
   await step('Build project', async () => {
+    // store cache to add after build directory is emptied
+    let cache = fs.readJsonSync(`${DIR}/build/cache.json`);
```

Comment:
Does this throw an error if `cache.json` or the `build` folder don't exist? I think the usual expectation is that you can delete auto-generated folder like `build` and everything should still keep working. Would be nice if we can also ensure that!